docs/source/release/v6.3.0/index.rst

```diff
@@ -1,22 +1,76 @@
+   :width: 350px
    :align: right
 
 .. contents:: Table of Contents
    :local:
 
-.. warning:: This release is still under construction. The changes can be found in the nightly builds on the `download page`_.
+Summary
+-------
 
 We are proud to announce version 6.3.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+Much of this version provides improvements for users, building on exisiting algorithms and GUIs.
+Plotting has also seen a number of updates and users will benefit from improved documentation too.
+
+In addition to many improvements we are delighted to announce some new features including:
+
+- The :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` now supports two texture groupings, ``Texture20`` and ``Texture30`` (see image above).
+- The Error Reporter can now remember and prefill the user's name and e-mail address.
+- The :ref:`InstrumentViewer` :ref:`Pick Tab<instrumentviewer_pick_tab>` has a new panel for allowing the rebinning of workspaces.
+- In the :ref:`Frequency Domain Analysis <Frequency_Domain_Analysis-ref>` and the :ref:`Muon Analysis <Muon_Analysis-ref>` GUIs the confidence interval of a fit, previously shown as an error bar, is now represented by a shaded region.
+- Several new fitting functions for Muons have been provided.
+
+
+Conda
+-----
+
+.. figure:: ../../images/conda_logo.svg
+   :align: right
+   :width: 400px
+
+   |copy| 2017 Continuum Analytics, Inc. (dba Anaconda, Inc.).
+
+   https://www.anaconda.com. All Rights Reserved
+
+
+We are excited to announce the release of brand-new packages built for the `conda package manager <https://docs.conda.io/en/latest/>`__.
+Up until this release only a single ``mantid-framework`` package has been available for Linux but
+this release sees full support for our standard platforms (Linux, Windows, macOS).
+Furthermore, the traditional monolithic package has been split into 3 packages in the conda world:
+
+- ``mantid``: Access to the algorithms and workspace from Python (no GUI components).
+
+- ``mantidqt``: Access to custom-made Qt widgets built on top of mantid functionality,
+  e.g. InstrumentViewer, SliceViewer.
+
+- ``mantidworkbench``: A package shippping the traditional Workbench application.
+
+For instructions on how to access these packages please see the `conda installation instructions <https://download.mantidproject.org/conda.html>`__.
+
+Future Packaging Changes
+------------------------
+
+In future releases we will be reworking how our installer packages are constructed so that we can replace an
+aging system that is hard to maintain and keep up to date.
+
+For Windows & macOS, users should see no change in how these packages function and are used. However, the version of Python bundled
+internally will start to change more regularly, at a frequency yet undefined, to keep up with external developments.
+
+For Linux we intend to primarily push users towards the conda distribution packages but we will work with users
```

Comment:
```suggestion
We planning on pushing Linux users towards the Conda distribution packages and dropping package (.deb and .rpm) builds in the future. This would Mantid easier to install on different distros (though support would be unofficial). But we're open to feedback and suggestions on this direction.
```

Reply:
@martyngigg are you ok with this wording?

Reply:
The start of the suggested wording is grammatically incorrect, and should be either:
- We plan to push Linux users...
- We are planning on pushing Linus users...

Reply:
I'm happy with the reworded paragraph but as Tom says the start should be "We plan to push..."

I can make this update when I make the tweak for adding an extra sentance higher up.